src/components/App/__test/App.spec.ts

```diff
@@ -3,15 +3,12 @@ import { render } from '@testing-library/svelte';
 
 describe('testing getDisplayDateTime', () => {
   it('it works', async () => {
```

Comment:
It already has "it" word, no need the "it" inside the brackets. I think you can do some thing like this
```suggestion
  it('should return datetime', async () => {
```

depends on the context